src/index.js

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable arrow-parens */
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-unused-vars */
+import _, { delay, slice } from 'lodash';
+import './style.css';
+import { checked, textDecorationChecked, changeState } from './checked.js';
+import { addTask, deleteTask, deleteAll } from './add-and-remove.js';
+
+const ul = document.querySelector('.task-list');
+
+function saveDataLocalStorage(activities) {
+  localStorage.setItem('activities', JSON.stringify(activities));
+}
+
+function loadDataLocalStorage() {
+  const activities = [];
+
+  return JSON.parse(localStorage.getItem('activities')) || activities;
+}
+
+function displayTasks() {
+  ul.innerHTML = '';
+  const activities = loadDataLocalStorage();
+
+  for (let i = 0; i < activities.length; i++) {
+    const li = document.createElement('li');
+    const checkIn = checked(activities, i);
```

Comment:
I suggest you use a higher-order method e.g (forEach) instead of the for loop as it makes your javaScript "KISS"